js/constants/config.js

```diff
@@ -7,7 +7,7 @@
 // https://vault-staging.brave.com for a dev build
 // http://localhost:3000 for production
 const vaultHost = process.env.VAULT_HOST || 'https://vault-staging.brave.com'
-const adHost = process.env.AD_HOST || '//cdn.brave.com'
+const adHost = process.env.AD_HOST || '//oip.brave.com'
```

Comment:
Cost and overhead I suppose. There is no assumption that the data is going to a trusted 3rd-party so the only reason we use https at all is to avoid mixed content.